backend/src/v5/middleware/dataConverter/inputs/sso/aad/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { createResponseCode, templates } = require('../../../../../utils/responseCodes');
+const axios = require('axios');
```

Comment:
should I replace httpsReq in v4 and make v5 use it ?

Reply:
no instead of importing v4 [here](https://github.com/3drepo/3drepo.io/blob/staging/backend/src/v5/utils/httpsReq.js) 

write the functions you need using axios (and make sure whatever else is relying on this file is changed appropriately)

Reply:
Maybe rename it to webRequests, we don't need to have httpsReq specifically, if we start calling http protocol it should live there too